pkg/discovery/discovery.go

```diff
@@ -243,6 +240,12 @@ func setStatus(client kubernetes.Interface, namespace string, status *pluginaggr
 		return errors.Wrap(err, "failed to marshal the patch")
 	}
 
-	_, err = client.CoreV1().Pods(namespace).Patch(pluginaggregation.StatusPodName, types.MergePatchType, patchBytes)
+	// Determine sonobuoy pod name
+	podName, err := pluginaggregation.GetStatusPodName(client, namespace)
+	if err != nil {
+		return errors.Wrap(err, "failed to get name of status pod")
```

Comment:
`status pod` I think is a little confusing.

Suggestion `failed to get the name of the aggregator pod to set the status on`

Reply:
There are a few other lines like this that I stopped commenting it on; fyi (so you dont just look at the lines I pointed out)